chapter2/tests/t1/chapter2-t1-two-files.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
You can add `set -eo pipefail` after `#!/bin/bash` line to immediately exit script if it has execution errors
https://distroid.net/set-pipefail-bash-scripts/